### PR TITLE
Release 7.9.1

### PR DESCRIPTION
**Pull request by tolusha**

### CODE REVIEW

File: olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/7.9.1/eclipse-che-preview-kubernetes.v7.9.1.clusterserviceversion.yaml.diff [hunk 1]
```diff
@@ -0,0 +1,70 @@
+--- /home/tolusha/gocode/src/github.com/eclipse/che-operator/olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/7.9.0/eclipse-che-preview-kubernetes.v7.9.0.clusterserviceversion.yaml	2020-02-26 09:42:38.504787505 +0200
++++ /home/tolusha/gocode/src/github.com/eclipse/che-operator/olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/7.9.1/eclipse-che-preview-kubernetes.v7.9.1.clusterserviceversion.yaml	2020-03-08 15:51:15.980971338 +0200
+@@ -19,7 +19,7 @@
+                 "cheImageTag": "",
+                 "devfileRegistryImage": "",
+                 "pluginRegistryImage": "",
+-                "tlsSupport": false,
++                "tlsSupport": true,
```
Comment (nickboldt) [inline olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/7.9.1/eclipse-che-preview-kubernetes.v7.9.1.clusterserviceversion.yaml.diff]: So we're turning https on by default in 7.9.1? Are we doing the same in 7.10?

File: olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/7.9.1/eclipse-che-preview-kubernetes.v7.9.1.clusterserviceversion.yaml.diff [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+-                  value: quay.io/eclipse/che-keycloak:7.9.0
++                  value: quay.io/eclipse/che-keycloak:7.9.1
+                 - name: IMAGE_default_che_workspace_plugin_broker_metadata
+                   value: quay.io/eclipse/che-plugin-metadata-broker:v3.1.1
+                 - name: IMAGE_default_che_workspace_plugin_broker_artifacts
+                   value: quay.io/eclipse/che-plugin-artifacts-broker:v3.1.1
+                 - name: IMAGE_default_che_server_secure_exposer_jwt_proxy_image
+-                  value: quay.io/eclipse/che-jwtproxy:dbd0578
+-                image: quay.io/eclipse/che-operator:7.9.0
++                  value: quay.io/eclipse/che-jwtproxy:810d89c
```
Comment (nickboldt) [inline olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/7.9.1/eclipse-che-preview-kubernetes.v7.9.1.clusterserviceversion.yaml.diff]: this change seems LONG overdude as dbd0578 is from Sept 5, but 810d89c is from Feb 26.

You sure we shouldn't use a newer build? https://quay.io/repository/eclipse/che-jwtproxy?tag=latest&tab=tags

File: olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/7.9.1/eclipse-che-preview-kubernetes.v7.9.1.clusterserviceversion.yaml.diff (hunk 1 above)
Comment (tolusha) [reply olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/7.9.1/eclipse-che-preview-kubernetes.v7.9.1.clusterserviceversion.yaml.diff]: sure

File: olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/7.9.1/eclipse-che-preview-kubernetes.v7.9.1.clusterserviceversion.yaml.diff (hunk 2 above)
Comment (tolusha) [reply olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/7.9.1/eclipse-che-preview-kubernetes.v7.9.1.clusterserviceversion.yaml.diff]: https://github.com/eclipse/che/blob/7.9.1/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L505

`che.server.secure_exposer.jwtproxy.image=quay.io/eclipse/che-jwtproxy:810d89c`

**Pull request merged**